### PR TITLE
change the default flush to cosole - tmp file

### PR DESCRIPTION
**Pull request by benmalef**
Change the default implementation.
Save the log in the tmp dir.

### CODE REVIEW

File: GANDLF/utils/gandlf_logger.py [hunk 1]
```diff
@@ -23,13 +24,21 @@ def _flush_to_console():
     logging.root.addHandler(console_handler)
 
 
+def _create_tmp_log_file():
+    tmp_dir = Path(tempfile.gettempdir())
+    log_dir = Path.joinpath(tmp_dir, "gandlf")
+    log_dir.mkdir(parents=True, exist_ok=True)
+    log_file = Path.joinpath(log_dir, "gandlf.log")
```
Comment (sarthakpati) [inline GANDLF/utils/gandlf_logger.py]: I think from a user standpoint, putting the logs in `${tmp_dir}/.gandlf/${timestamp}.log` makes more sense (and it is also a more standardized approach to dealing with user-level logs):

```suggestion
    log_dir = Path.joinpath(tmp_dir, ".gandlf")
    log_dir.mkdir(parents=True, exist_ok=True)
    log_file = Path.joinpath(log_dir, get_unique_timestamp() + ".log")
```

File: GANDLF/utils/gandlf_logger.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 from importlib import resources
 import colorlog
+import tempfile
```
Comment (sarthakpati) [inline GANDLF/utils/gandlf_logger.py:6]: Added import.

```suggestion
import tempfile
from GANDLF.utils import get_unique_timestamp
```

File: GANDLF/entrypoints/cli_tool.py [hunk 3]
```diff
@@ -25,7 +25,7 @@ def gandlf(ctx, loglevel):
     ctx.ensure_object(dict)
     ctx.obj["LOGLEVEL"] = loglevel
     # setup_logging(loglevel)
-    gandlf_logger_setup()
+    # gandlf_logger_setup()
```
Comment (sarthakpati) [inline GANDLF/entrypoints/cli_tool.py]: Why is this commented?

**Comment by sarthakpati** [comment]
Right now, the only issue is that the code has not been linted, so just run `black .` after accepting my edits and it should be fine. 😄

**Comment by benmalef** [comment]
Thanks a lot @sarthakpati

**Pull request merged**